doc/TopicModelling.md

```diff
@@ -0,0 +1 @@
+The topic modeling analyzer is an analyzing system that takes keywords and implements them into a graph that demonstrates the frequency they were used from each person. It doesn't take the number of times those keywords were used but rather if they were used from certain users. If one of the keywords was used it will be put into either a histogram graph or a scatter chart. You can also change the number of topics and the amount of words per topic. For instance topic 0 has the keywords design, java, and people while topic 3 has robot, race, and ethical. The topics are also separated by which assignment they are analyzing which give us a better feel of what it is that's being calculated, and where those keywords originate from.
```

Comment:
This is probably not what topic modeling actually does. Check this link: https://www.machinelearningplus.com/nlp/topic-modeling-gensim-python/.

Also, I think this applies to all: the examples we use (robot, race, ethical, etc.) probably shouldn't go into the description of the page. I imagine when users enter their own input, these would not make sense.